data/en/test_sum_equal_or_less_validation_against_total.json

```diff
@@ -3,7 +3,7 @@
     "schema_version": "0.0.1",
     "data_version": "0.0.2",
     "survey_id": "0",
-    "title": "Grouped validation against total test survey",
+    "title": "A test schema for LESS THAN or EQUAL validation of a sum of answers against a given total",
     "theme": "default",
```

Comment:
Bit picky but not sure if that is still clear. What about:
`A schema to test a sum of answers are Less Than or Equals to a given total`?